include/RAJA/policy/cuda/WorkGroup/WorkRunner.hpp

```diff
@@ -87,6 +87,62 @@ struct WorkRunner<
   }
 };
 
+/*!
+ * Explicit BLOCKS_PER_SM version.
+ * Runs work in a storage container in order
+ * and returns any per run resources
+ */
+template <size_t BLOCK_SIZE, size_t BLOCKS_PER_SM, bool Async,
+          typename ALLOCATOR_T,
+          typename INDEX_T,
+          typename ... Args>
+struct WorkRunner<
+        RAJA::cuda_work_explicit<BLOCK_SIZE, BLOCKS_PER_SM, Async>,
```

Comment:
I think the RAJA::cuda_work_explicit specialization should also handle RAJA::cuda_work, so we can keep just one set of specializations.

Reply:
Forgot to remove some of these, good catch!